libs/hls.liq

```diff
@@ -86,7 +86,7 @@ def input.hls(~id="",~reload=10.,uri)
 end
 
 # @flag hidden
-def output.harbor.hls.base(~id,~fallible,~on_start,~on_stop,~streams_info,
+def output.harbor.hls_base(~id,~fallible,~on_start,~on_stop,~streams_info,
```

Comment:
Does the `hidden` flag apply to the `output.harbor` type? Is it carried over to prevent displaying it in the documented type?

Reply:
`hidden` applies to `output.harbor.hls_base`. When we register a function in the doc, we register it as it is declared (with all the dots and so on). We don't register the intermediate records. I don't think the flags are carried over (didn't change anything on this side): `hidden` simply means that *this* definition won't occur in the doc.